src/sparql/reports/COUNT-all_disease_excluding_susceptibility.sparql

```diff
@@ -0,0 +1,23 @@
+PREFIX rdfs: <http://www.w3.org/2000/01/rdf-schema#>
+PREFIX owl: <http://www.w3.org/2002/07/owl#>
+PREFIX obo: <http://purl.obolibrary.org/obo/>
+
+
+# Count of all non-obsolete 'disease' classes excluding 'disease susceptibility' subclasses
+
+SELECT (COUNT(DISTINCT ?class) AS ?countAllDiseaseExcludingSusceptibility)
+WHERE {
+  {
+    SELECT DISTINCT ?class WHERE {
+      ?class rdfs:subClassOf* obo:MONDO_0000001 .
+      FILTER(STRSTARTS(STR(?class), "http://purl.obolibrary.org/obo/MONDO_"))
+      FILTER NOT EXISTS { ?class owl:deprecated true }
+    }
+  }
+  MINUS {
+    SELECT DISTINCT ?class WHERE {
+      # MONDO:0042489 'disease susceptibility'
+      ?class rdfs:subClassOf* obo:MONDO_0042489 .
+    }
+  }
```

Comment:
Nico - from Tech call, this `MINUS` statement should be removed from ALL queries.